### PR TITLE
Bump react-native to 0.73.2

### PR DESCRIPTION
**Pull request by jkhusanov**
Updated React Native version in the example app to the latest

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -1,150 +1,87 @@
...
-def nativeArchitectures = project.getProperties().get("reactNativeDebugArchitectures")
-
 android {
     ndkVersion rootProject.ext.ndkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdk rootProject.ext.compileSdkVersion
 
-    compileSdkVersion rootProject.ext.compileSdkVersion
-
+    namespace "com.exampleapp"
```
Comment (calebmackdavenport) [inline example/android/app/build.gradle:77]: Lol this threw me off. I see now that it's your example app within this repo

File: example/.gitignore [hunk 2]
```diff
@@ -29,32 +31,36 @@ build/
...
+**/fastlane/screenshots
+**/fastlane/test_output
 
 # Bundle artifact
 *.jsbundle
 
-# CocoaPods
+# Ruby / CocoaPods
 /ios/Pods/
+/vendor/bundle/
```
Comment (calebmackdavenport) [inline example/.gitignore:61]: We could also add `Gemfile.lock` to this example project's gitignore and the `yarn bundle:install` to the package.json

**Review by calebmackdavenport — APPROVED** [review]
Overall this looks good. Left a few small notes if you want to consider them.

At the moment, I can't build iOS right now but it appears unrelated to this unit of work, more widespread for any debug build across any app on my machine.

Android builds well, works well.

My initial concern was around, if adopting this update would require neutron to be using Java 17 like we ran into with persona. That doesn't appear to be the case thankfully.

**Comment by jkhusanov** [comment]
Having build issues, going to bump it again later.

**Comment by jkhusanov** [comment]
Closing this PR as the newer version has been introduced in PR https://github.com/shipt/segmented-arc-for-react-native/pull/71

**Pull request closed without merging**